cmd/cephcsi.go

```diff
@@ -78,7 +78,7 @@ func init() {
 	flag.DurationVar(&conf.PollTime, "polltime", time.Second*pollTime, "time interval in seconds between each poll")
 	flag.DurationVar(&conf.PoolTimeout, "timeout", time.Second*probeTimeout, "probe timeout in seconds")
 
-	flag.BoolVar(&conf.EnableGRPCMetrics, "enablegrpcmetrics", false, "[DEPRECATED] enable grpc metrics")
+	flag.BoolVar(&conf.EnableGRPCMetrics, "enablegrpcmetrics", true, "[DEPRECATED] enable grpc metrics")
```

Comment:
I understand that this is used for starting the http service. But it probably should be a different commit (and mention that the grpc metrics will be removed).

Reply:
sure

Reply:
@yati1998 why we are flipping it back ?

Reply:
disable by default.

Reply:
it is disabled by default in the current state